crypto/bls12381/key_bls12381.go

```diff
@@ -103,7 +101,7 @@ func (PrivKey) Type() string {
 
 // Sign signs the given byte array.
 func (privKey PrivKey) Sign(msg []byte) ([]byte, error) {
-	signature := new(blstSignature).Sign(privKey.sk, msg, dstMinSig)
+	signature := new(blstSignature).Sign(privKey.sk, msg, dstMinPk)
```

Comment:
This should be fixed on main cometbft too!

Reply:
I think for now it is worth opening an issue on the CometBFT repo until we see how to proceed.

Reply:
https://github.com/cometbft/cometbft/issues/4783